pkg/cfg/queries.yaml

```diff
@@ -0,0 +1,11 @@
+valuesets:
```

Comment:
Put this into testdata, add another table to the test data, tableId: snomed with query:
```
 - tableId: snomed
    queries:
      - query: "select concept_id,concept_name,domain_id,vocabulary_id,concept_class_id,concept_code from concept where vocabulary_id='SNOMED' and concept_code=%(code)s and valid_end_date='20991231';"
        columns: 
              - code